statedb/treeupdate.go

```diff
@@ -314,7 +313,7 @@ func (t *TreeTx) Commit(version uint32) error {
 	if err := t.CommitOnTx(version); err != nil {
 		return err
 	}
-	return t.tx.Commit()
+	return t.SaveWithoutCommit()
```

Comment:
Can you explain this please?

Reply:
leftover from statedb rabbit hole.
 TBH, it's equivalent, `t.SaveWithoutCommit` does exactly `t.tx.Commit`
https://github.com/vocdoni/vocdoni-node/blob/d50e8db56ace073bcc81ed2aa3067b848ff81039/statedb/treeupdate.go#L352-L354

but i'll revert this line, no need to touch this at all in fact

Reply:
fixed in 7f0c9aa